src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -18,23 +18,71 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       show: false,
     },
     tooltip: {
-      format: {
-        title: (date) => {
-          return intl.formatDate(date, {
+      contents: function(data) {
+        if (data.length) {
+          const date = intl.formatDate(data[0].x, {
             timeZone: 'UTC',
             day: 'numeric',
             month: 'long',
             year: 'numeric',
             weekday: 'long',
           });
-        },
-        value: (value) => {
-          return (
-            intl.formatMessage({
-              id: 'reports_daily_visits.tooltip_page_views',
-            }) + value
-          );
-        },
+          let quantity = '',
+            withEmail = '',
+            withoutEmail = '';
+          for (const item of data) {
```

Comment:
This structure is difficult to follow. 

What is Data?

If it is an array, what do you think about using _reduce_ as @fcoronelmakingsense did?

What happens if there are more than a _quantity_ item or less than one?

Reply:
There will always be three, this is associated to the columns defined in ln 129. 
```
 keys: {
              x: 'from',	              x: 'from',
              value: ['quantity'],	              value: ['quantity', 'withEmail', 'withoutEmail'],
            },
```

Reply:
It will try to use reduce here.